src/main/java/com/treasure/hunt/view/settings/SettingsWindow.java

```diff
@@ -14,17 +15,16 @@
 @Slf4j
 public class SettingsWindow {
 
-    public static void show() throws Exception {
+    public static void show() {
         Stage stage = new Stage();
-        final FXMLLoader fxmlLoader = new FXMLLoader(Main.class.getResource("/layout/settings/settings.fxml"));
-        Scene scene = new Scene(fxmlLoader.load());
+        final Widget<SettingsController, Region> settings = new Widget<>("/layout/settings/settings.fxml");
```

Comment:
Why is that used here Widget was nt intended for this use if u ask me

Reply:
Widget is a class generic enough to be used in this way. I thought DRY code would fit this case better